<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPacketSenderUI</class>
 <widget class="QGroupBox" name="CPacketSenderUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QTextEdit" name="teSendPackets">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>391</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSend">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSpam">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbTime">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string>ms</string>
   </property>
   <property name="minimum">
    <number>50</number>
   </property>
   <property name="maximum">
    <number>2000000000</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="lTime">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>152</y>
     <width>38</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Time:</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pbSend</sender>
   <signal>clicked()</signal>
   <receiver>CPacketSenderUI</receiver>
   <slot>onSendPacket()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbSpam</sender>
   <signal>clicked()</signal>
   <receiver>CPacketSenderUI</receiver>
   <slot>onSpamPacket()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSendPacket()</slot>
  <slot>onSpamPacket()</slot>
 </slots>
</ui>
